plane_inspection_node:
  ros__parameters:
    world_frame: world_ned
    robot_frame: girona500/base_link
